### DISCUSSION TITLE
Does manim really support Text element in svg?

### DISCUSSION THREAD

**Discussion by MinuteSphinxE** [3b1b/manim#1811] [Q&A]
![fdc23206a83d86eed3e5f0c0d80e195](https://user-images.githubusercontent.com/104899078/168328057-edcecc3a-622c-4614-9f6c-63618df39ae6.png)

I tried to add some svg into this, but it only render the lines and polygon
![55ff925d8a40b7540626741a57ca378](https://user-images.githubusercontent.com/104899078/168327357-0e27082c-bab1-4116-80d5-2680ef1a248f.png)
it should be like this
![cdf83bdbd1f810d1253c527201c7f46](https://user-images.githubusercontent.com/104899078/168327426-63e81349-de18-4336-a8e6-3e11c1cbd66e.png)
the text was missing. then i found the svg file has elements of "Text", I wonder if the svg render engine "svgelements" does not support text.
but it's not. It can render svgs.
Is there any solutions ..?

**Comment by AlexisTercero55** [comment]
today 26/12/22 has no solution yet, im using ManimCE. :(

**Reply by greycodee** [reply]
2023/02/04
os: ArchLinux
not yet

```bash
╰─$ manimgl main.py TextMoreCustomization                                                               130 ↵
ManimGL v1.6.1
[15:57:00] INFO     Using the default configuration file, which you can modify in                config.py:265
                    `/home/zheng/.conda/envs/manim-env/lib/python3.10/site-packages/manimlib/def              
                    ault_config.yml`                                                                          
           INFO     If you want to create a local configuration file, you can create a file      config.py:266
                    named `custom_config.yml`, or run `manimgl --config`                                      
           WARNING  Unsupported element type: <class 'svgelements.svgelements.Use'>         svg_mobject.py:174
           WARNING  Unsupported element type: <class 'svgelements.svgelements.Use'>         svg_mobject.py:174
```

**Reply by TonyCrane** [reply]
> 2023/02/04 os: ArchLinux not yet
> 
> ```shell
> ╰─$ manimgl main.py TextMoreCustomization                                                               130 ↵
> ManimGL v1.6.1
> [15:57:00] INFO     Using the default configuration file, which you can modify in                config.py:265
>                     `/home/zheng/.conda/envs/manim-env/lib/python3.10/site-packages/manimlib/def              
>                     ault_config.yml`                                                                          
>            INFO     If you want to create a local configuration file, you can create a file      config.py:266
>                     named `custom_config.yml`, or run `manimgl --config`                                      
>            WARNING  Unsupported element type: <class 'svgelements.svgelements.Use'>         svg_mobject.py:174
>            WARNING  Unsupported element type: <class 'svgelements.svgelements.Use'>         svg_mobject.py:174
> ```

this is not caused by not supporting <text\> element. See https://github.com/3b1b/manim/issues/1904#issuecomment-1364468069 for solution.
